src/transactions/test/TxEnvelopeTests.cpp

```diff
@@ -1886,6 +1886,7 @@ TEST_CASE_VERSIONS("txenvelope", "[tx][envelope]")
                                      .header.scpValue.closeTime;
             app->getHerder().externalizeValue(txSet, 3, lastCloseTime,
                                               emptyUpgradeSteps);
+            testutil::crankFor(app->getClock(), std::chrono::seconds(1));
```

Comment:
The gist is that externalize flow is now asynchronous, so by calling plain externalizeValue we won't actually apply anything until we crank a little bit. That being said, I think this line is a remnant from when I ran the whole test suite with parallel ledger apply; since it's off by default now, this line can probably be removed (for now)